modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -1,24 +1,34 @@
-:navtitle: Adding a Visual Studio Code Extension
+:navtitle: Adding a Visual Studio Code extension to a workspace
 :keywords: Visual Studio Code extension, user-guide
 
 [id="adding-visual-studio-code-extension_{context}"]
-= Adding a Visual Studio Code Extension
+= Adding a Visual Studio Code extension to a workspace
 
-IDE specific plug-ins are no longer part of devfile with devfile v2. Instead, it uses the specific meta-folder to describe the plug-ins.
+Devfiles in the devfile v2 format don't contain any IDE-specific plug-ins. Instead, the plug-ins are described in a specific meta-folder.
 
-[id="visual-studio-code-extensions-json"]
-== Using `.vscode/extensions.json`
+[id="plug-in-registries"]
+== {prod-short} plug-in registries overview
 
-The easiest way to add a Visual Studio Code extension is to add the extension to the `.vscode/extensions.json` file. Using Che-Theia, the extension will be automatically installed and configured. With Che-Code, the Visual Studio Code fork, at the startup of the workspace, Visual Studio Code shows a recommendation pop-up to install the extension.
+{prod-short} has a default plug-in registry installed in every {prod-short} instance. The Che-Theia IDE searches and installs the extensions from that location. 
 
-.Procedure
+Check this {prod-short} https://github.com/eclipse-che/che-plugin-registry[registry project] for an overview of default plug-ins and source code. An online instance that is refreshed after every commit to the main branch, is located https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/[here]. You can use a different plug-in registry for Che-Theia if you don't work in air gap mode: only the default registry is available there.
+
+The plug-in overview for Che-Code Visual Studio Code editor is located in the https://www.open-vsx.org/[OpenVSX instance]. Air gap mode is not yet supported for this editor.
```

Comment:
The plug-in and extension overview for Che-Code Visual Studio Code editor is located in the https://www.open-vsx.org/[OpenVSX instance]. Air gap mode is not yet supported for this editor.